Components/alert.css

```diff
@@ -0,0 +1,25 @@
+.alert{
+    margin: 1rem 0rem;
+    padding: 1rem 1rem;
+    border-radius: 4px;
+}
+.alert-primary{
+    background-color: rgb(101, 229, 252);
+    color: blue;
+}
+.alert-secondary{
+    background-color: rgb(223, 217, 217);
```

Comment:
As you are using CSS variables for naming color. So, it would be good if you will keep that consistent.

Reply:
yup will implement this as well.